src/main/resources/swagger/api-docs.yaml

```diff
@@ -9494,6 +9547,15 @@ components:
           description: The run cost of this workflow
           format: float
       description: ""
+    UserCommentUpdateOperation:
+      required:
+        - userComment
+      type: object
+      properties:
+        userComment:
+          type: string
+          description: The updated value for user comment (max length 1000 characters)
```

Comment:
Is it worth mentioning this 1000 character limit in the docs for the initial submission endpoint too?